MSBuildLaunchPad/ParserFactory.cs

```diff
@@ -4,15 +4,15 @@ namespace Lextm.MSBuildLaunchPad
 {
     public static class ParserFactory
     {
-        public static IParser Parse(string fileName)
+        public static IParser ParseFile(string fileName)

```

Comment:
I am worried that this method could have performance issues. Maybe we can write some performance tests and see total time of execution?